docker-compose.yml

```diff
@@ -181,7 +183,7 @@ services:
     volumes:
       - ./dataland-frontend:/app
       - frontend-dev-node-modules:/app/node_modules
-    command: sh -c "cd /app && npm install && npm run dev"
+    command: sh -c "cd /app && npm install && npm install --no-save @esbuild/linux-x64 && npm run dev"
```

Comment:
why do we need that? Why isn't it part of the package.json?

Reply:
I believe this is a workaround to make the stack work on linux as well. If I recall correctly the issue is that in the package-lock.json you can only have either the windows or unix version of this package. To work around that it is added here manually. For more details you need to ask @MarcTM01

Reply:
Can you please revert this? I don't understand yet why we need that - and I feel this is not the right fix for whatever problem.

Reply:
Removed

Reply:
ok